samples/MusicStore/ForTesting/Mocks/StartupSocialTesting.cs

```diff
@@ -94,7 +94,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.AddPolicy("ManageStore", new AuthorizationPolicyBuilder().RequireClaim("ManageStore", "Allowed").Build());
             });
 
-            services.AddFacebookAuthentication(options =>
+            services.AddAuthentication().AddFacebook(options =>
```

Comment:
Nit: we typically add a newline before each of the `.AddXXX` call that hang off builders like .AddAuthentication